test/testcafe/framework/page-objects/ChallengeFactorPageObject.js

```diff
@@ -26,7 +26,11 @@ export default class ChallengeFactorPageObject extends BasePageObject {
     return Selector(SWITCH_AUTHENTICATOR_SELECTOR).textContent;
   }
 
-  clickNextButton() {
+  clickNextButton(name) {
+    if (userVariables.v3) {
+      return this.form.clickSaveButton(name);
+    }
+    
     return this.form.clickSaveButton();
   }
```

Comment:
Wouldn't this work for both v2 and v3?
```suggestion
  clickNextButton(name) {
    return this.form.clickSaveButton(name);
  }
```